test/smoke/src/areas/positron/plots/plots.test.ts

```diff
@@ -283,6 +283,101 @@ plt.show()`;
 
 				await app.workbench.positronPlots.waitForNoPlots();
 			});
+
+			it('Python - Verifies bqplot Python widget [C720869]', async function () {
```

Comment:
Since the code for each of these test cases is the same, should we extract and reuse a function?

Reply:
Good idea.  Done!